addons/l10n_co_pos_details/views/templates.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="internal_layout" inherit_id="web.internal_layout" primary="True">
+            <xpath expr="//span[@t-esc='company.name']" position="after">
+                <t t-esc="company.vat"/>
+            </xpath>
+        </template>
+
+        <template id="report_saledetails" inherit_id="point_of_sale.report_saledetails" primary="True">
+            <xpath expr="//t[@t-call='web.internal_layout']" position="attributes">
+                <attribute name="t-call">l10n_co_pos_details.internal_layout</attribute>
+            </xpath>
+            <xpath expr="//div[hasclass('page')]/div/h2" position="before">
+                <h2 t-if="pos_config"><t t-esc="pos_config.name"/></h2>
+            </xpath>
+            <xpath expr="//div[hasclass('page')]/h3[1]" position="attributes">
+                <attribute name="t-if">include_products</attribute>
+            </xpath>
+            <xpath expr="//div[hasclass('page')]/table[1]" position="attributes">
+                <attribute name="t-if">include_products</attribute>
+            </xpath>
+            <xpath expr="//div[hasclass('page')]/table[2]/thead/tr/th[1]" position="after">
```

Comment:
Is there no way to make the xpaths a little less complicated?  This risks breaking pretty fast.

Reply:
@bat-odoo

Reply:
@jco-odoo This is the only possible way to make xpath, in another way we need to add class in standard then we can use that class to make xpath.